.vscode/tasks.json

```diff
@@ -0,0 +1,83 @@
+{
+  "version": "2.0.0",
+  "tasks": [
+    {
+      "label": "Develop Frontend",
+      "type": "gulp",
+      "task": "develop-app",
+      "problemMatcher": {
+        "base": "$tsc",
```

Comment:
This is not true… but it works! 😬 

I had to define a full problem matcher but only care about the background part. Not sure how to work around and docs are unclear.